Products/zms/zms.py

```diff
@@ -224,19 +230,21 @@ def manage_addZMS(self, lang, manage_lang, REQUEST, RESPONSE):
     obj = initZMS(homeElmnt, 'content', titlealt, title, lang, manage_lang, REQUEST)
     
     ##### Add Theme ####
-    themeId = importTheme(obj,REQUEST['theme'])
-    obj.setConfProperty('ZMS.theme',themeId)
+    if REQUEST.get('theme','conf:acquire') != 'conf:acquire':

```

Comment:
should define a constant for 'conf:acquire' somewhere

Reply:
withjout refactoring a constant is just another name for a name. And it will get more difficult to find the code locations that have to be refactored. After centralizing / getting rid theme.zcml it definitely will make sense.